products/extensions/b2b-components/README.md

```diff
@@ -0,0 +1 @@
+# B2B components
```

Comment:
Please provide a short description of what are B2B components .

Reply:
B2B components are all B2B features, which can be activated and deactivated individually.
So idk what I could write about this.... :shrug: 
Would that be enough?:
`Shopware B2B Components are all B2B features, which can be activated and deactivated individually.`
The concept of the feature enabling and disabling is currently under construction, so I can't describe it more detailed :shrug: